test/integration/running/multi_inst_app_no_init/ie/mi_app/instances.yml

```diff
@@ -0,0 +1,4 @@
+router:                                                                                         
```

Comment:
AFAIU "mi" == "multi instalce", but what is "ie"?

Reply:
Instances enabled. Or Internet Explorer O_o
Renamed.